portfolio/src/main/webapp/script.js

```diff
@@ -72,17 +72,32 @@ function createListElement(comment) {
     .then((response) => response.text())
     .then((data) => {
       liElement.innerHTML = data;
-      liElement.getElementsByClassName("comment-words")[0].innerText = comment.content;
-      //Add funcitonality to the delete button
-      deleteButtonElement = liElement.getElementsByClassName("comment-delete")[0];
+      liElement.getElementsByClassName("comment-words")[0].innerText =
+        comment.content;
+      //Add functionality to the delete button
```

Comment:
we can get rid of this comment, since it doesn't really say anything that the code doesn't already say.

Reply:
done